frontend/src/context/layers/composite_data.ts

```diff
@@ -13,26 +14,25 @@ export const fetchCompositeLayerData: LazyLoader<CompositeLayerProps> = () => as
   params: LayerDataParams<CompositeLayerProps>,
   { dispatch },
 ) => {
-  // to complete later with new endpoint for composite chart
-
   const { layer, date } = params;
-  const endDate = (date ? new Date(date) : new Date())
-    .toISOString()
-    .split('T')[0];
-  const { baseUrl, inputLayers, startDate } = layer;
+  const startDate = date ? new Date(date) : new Date();
+  const endDate = new Date(startDate);
+  endDate.setMonth(endDate.getMonth() + 1);
+
+  const { baseUrl, inputLayers } = layer;
   const { boundingBox } = appConfig.map;
 
   // docs: https://hip-service.ovio.org/docs#/default/run_q_multi_geojson_q_multi_geojson_post
   const body = {
-    begin: startDate,
-    end: endDate,
+    begin: getFormattedDate(startDate, 'default'),
+    end: getFormattedDate(endDate, 'default'),
     area: {
       min_lon: boundingBox[0],
       min_lat: boundingBox[1],
       max_lon: boundingBox[2],
       max_lat: boundingBox[3],
       start_date: '2002-01-01',
-      end_date: endDate,
+      end_date: '2021-07-31',
```

Comment:
Are you sure this should be fixed?

Reply:
If yes, then it should probably be configured inside the layers.json

Reply:
and used similarly to `areaStartDate`

Reply:
Yea, I agree with Eric — the end date management should be pulled from the layer.json. I can shift it over now. 

I'll leave the fallback start date at '2002-01-01', I'm not confident I know what to test to ensure they're aren't regressions from removing it

Reply:
@lowriech, could you weigh in on the ETA on the hip_service side? Feel free to leverage me if I can useful in that

Reply:
@ericboucher @gislawill  hip_service is good to go on this.  It just responds to the config.  And having it be more dynamic poses some issues, namely that it won't be able to take advantage of the cache.